minari/__init__.py

```diff
@@ -36,4 +36,4 @@
     "split_dataset",
 ]
 
-__version__ = "0.3.1"
+__version__ = "0.4.0"
```

Comment:
We need to bump the minari version to the next release for the tests to download the compatible datasets with the current changes. This is the next Minari release correct? @balisujohn and @younik

Reply:
I think it's fine for this to be the release PR. Should we merge https://github.com/Farama-Foundation/Minari/pull/111 first?

Reply:
The test are missing but I think we can merge it now and I'll be working on the tests now.

Reply:
I think it is also fine to merge this PR (and bump the version even if it is not the last commit before releasing)